dev-support/test-patch.sh

```diff
@@ -2154,26 +2049,10 @@ function modules_workers
       continue
     fi
 
-    case ${BUILDTOOL} in
-      maven)
-        #shellcheck disable=SC2086
-        echo_and_redirect "${PATCH_DIR}/${repostatus}-${testtype}-${fn}.txt" \
-          ${MVN} "${MAVEN_ARGS[@]}" \
-            "${@//@@@MODULEFN@@@/${fn}}" \
-             ${MODULEEXTRAPARAM[${modindex}]//@@@MODULEFN@@@/${fn}} -Ptest-patch
-      ;;
-      ant)
-        #shellcheck disable=SC2086
-        echo_and_redirect "${PATCH_DIR}/${repostatus}-${testtype}-${fn}.txt" \
-          "${ANT}" "${ANT_ARGS[@]}" \
-          ${MODULEEXTRAPARAM[${modindex}]//@@@MODULEFN@@@/${fn}} \
-          "${@//@@@MODULEFN@@@/${fn}}"
-      ;;
-      *)
-        yetus_error "ERROR: Unsupported build tool."
-        return 1
-      ;;
-    esac
+    echo_and_redirect "${PATCH_DIR}/${repostatus}-${testtype}-${fn}.txt" \
```

Comment:
shellcheck: 7: note: Double quote to prevent globbing and word splitting. [SC2086]